src/main/java/seedu/address/model/person/Amount.java

```diff
@@ -0,0 +1,60 @@
+package seedu.address.model.person;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.commons.util.AppUtil.checkArgument;
+
+/**
+ * Represents a Payer paid amount in the address book.
+ */
+public class Amount {
+
+    public static final String MESSAGE_ADDRESS_CONSTRAINTS =
+            "Payer amount can only take in numerical number with 2 decimals, and it should not be blank";
+
+    /*
+     * The first character of the address must not be a whitespace,
+     * otherwise " " (a blank string) becomes a valid input.
+     */
+    public static final String MESSAGE_AMOUNT_CONSTRAINTS =
+            "Amount value can only contain numbers and comma when value is below 1";
+    public static final String AMOUNT_VALIDATION_REGEX = "^-?\\d*\\.\\d{2}$\n";
+
+    public final String value;
```

Comment:
Why storing the value in amount in String, instead of a double ? We are going to make some computations with these amount later.